src/room.py

```diff
@@ -6,10 +6,18 @@ class Room:
     def __init__(self, name, description):
         self.name = name
         self.description = description
+        self.items = []
```

Comment:
Great job using a list. We could take it one more step here and call this "contents". What else can be in rooms besides items?